### PR TITLE
Update patch plugin versions in gemfile lock for 6.8.13

### PR DESCRIPTION
**Pull request by andsel**

### CODE REVIEW

**Review by jsvd — CHANGES_REQUESTED** [review]
we should only include critical bug fixes and security patches in the 6.8 updates. I suggest reverting all the dependency bumps of libraries in this PR, and then manually checking if any of the plugin version bumps is a critical  fix that needs to go into it.

File: Gemfile.jruby-2.5.lock.release [hunk 1]
```diff
@@ -213,7 +212,7 @@ GEM
       elasticsearch (>= 5.0.3)
       logstash-core-plugin-api (>= 1.60, <= 2.99)
       manticore (~> 0.6)
-    logstash-filter-fingerprint (3.2.1)
+    logstash-filter-fingerprint (3.2.2)
```
Comment (andsel) [inline Gemfile.jruby-2.5.lock.release:215]: `3.2.2` includes a fix for consistent firgerprint in hash https://github.com/logstash-plugins/logstash-filter-fingerprint/pull/55

File: Gemfile.jruby-2.5.lock.release [hunk 2]
```diff
@@ -304,7 +303,7 @@ GEM
       logstash-core-plugin-api (>= 1.60, <= 2.99)
       rufus-scheduler
       stud (~> 0.0.22)
-    logstash-input-file (4.1.17)
+    logstash-input-file (4.1.18)
```
Comment (andsel) [inline Gemfile.jruby-2.5.lock.release:306]: Fix: sincedb_clean_after not being respected https://github.com/logstash-plugins/logstash-input-file/pull/276

File: Gemfile.jruby-2.5.lock.release [hunk 3]
```diff
@@ -416,7 +415,7 @@ GEM
     logstash-input-unix (3.0.7)
       logstash-codec-line
       logstash-core-plugin-api (>= 1.60, <= 2.99)
-    logstash-mixin-aws (4.3.0)
+    logstash-mixin-aws (4.4.1)
```
Comment (andsel) [inline Gemfile.jruby-2.5.lock.release:418]: Fixes on proxy settings https://github.com/logstash-plugins/logstash-mixin-aws/pull/50 https://github.com/logstash-plugins/logstash-mixin-aws/pull/48

File: Gemfile.jruby-2.5.lock.release [hunk 4]
```diff
@@ -516,9 +515,9 @@ GEM
     lru_redux (1.1.0)
     mail (2.6.6)
       mime-types (>= 1.16, < 4)
-    manticore (0.6.4-java)
+    manticore (0.7.0-java)
```
Comment (andsel) [inline Gemfile.jruby-2.5.lock.release:518]: - Fix a thread safety issue with regards to adding requests to the parallel execution queue while the client is already processing a queue  
- drop support for Ruby 1.7

**Comment by andsel** [comment]
These changes are all minors, so we close this bump PR

**Pull request closed without merging**